torch/distributed/rpc/api.py

```diff
@@ -235,12 +233,12 @@ def rpc_async(to, func, args=None, kwargs=None):
     Make a non-blocking RPC call to run function ``func`` on worker ``to``. RPC
     messages are sent and received in parallel to execution of Python code. This
     method is thread-safe. This method will immediately return a
-    torch.distributed.FutureMessage that can be awaited on.
+    ``torch.distributed.FutureMessage`` that can be awaited on.
 
     Arguments:
-        to (int or str): id or name of the destination worker.
+        to (int or str or WorkerInfo): id or name of the destination worker.
```

Comment:
It's prior to this PR, but it seems this cannot be int, as the API is not exposed in `RpcAgent`.